UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -281,6 +281,36 @@ public void CmdDropAllItems(uint itemSlotID, Vector3 Target)
 		}
 	}
 
+	/// <summary>
+	/// Transfers x amount of items from one hand to another. For stackable items only
+	/// </summary>
+	[Command]
+	public void CmdSplitStack(uint fromSlotID, NamedSlot fromSlot, int amountToTransfer)
+	{
+		if (fromSlot != NamedSlot.leftHand && fromSlot != NamedSlot.rightHand) return; //Only allowed to transfer from one hand to another
+		if (!Validations.CanInteract(playerScript, NetworkSide.Server, allowCuffed: false)) return; //Not allowed to transfer while cuffed
+		if (!Cooldowns.TryStartServer(playerScript, CommonCooldowns.Instance.Interaction)) return;
+
+		NamedSlot toSlot = PlayerManager.LocalPlayerScript.DynamicItemStorage.GetActiveHandSlot().NamedSlot.GetValueOrDefault(NamedSlot.none); //Were assuming that slot to which player wants to transfer stuff is always active hand
+		uint toSlotID = PlayerManager.LocalPlayerScript.DynamicItemStorage.GetActiveHandSlot().ItemStorage.gameObject.NetId();
+
+		var ObjFS = NetworkIdentity.spawned[fromSlotID].gameObject;
```

Comment:
```suggestion
                if(NetworkIdentity.spawned.TryGetValue(fromSlotID, out var objFS) == false) return;
```
camel case objFS

Reply:
This code fails cause objFS is network enitity and GetNamedItemSlot requires gameobject. May I revert back to original?